backend/src/main/java/org/cryptomator/hub/api/AuditLogResource.java

```diff
@@ -56,10 +61,11 @@ public List<AuditEventDto> getAllEvents(@QueryParam("startDate") Instant startDa
 	}
 
 	@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
-	@JsonSubTypes({
-			@JsonSubTypes.Type(value = CreateVaultEventDto.class, name = CreateVaultEvent.TYPE),
-			@JsonSubTypes.Type(value = UnlockVaultEventDto.class, name = UnlockVaultEvent.TYPE),
-			@JsonSubTypes.Type(value = UpdateVaultMembershipEventDto.class, name = UpdateVaultMembershipEvent.TYPE)
+	@JsonSubTypes({ //
+			@JsonSubTypes.Type(value = CreateVaultEventDto.class, name = CreateVaultEvent.TYPE), //
+			@JsonSubTypes.Type(value = UnlockVaultEventDto.class, name = UnlockVaultEvent.TYPE), //
+			@JsonSubTypes.Type(value = UpdateVaultEventDto.class, name = UpdateVaultEvent.TYPE), //
+			@JsonSubTypes.Type(value = UpdateVaultMembershipEventDto.class, name = UpdateVaultMembershipEvent.TYPE) //
```

Comment:
Here the Subtypes for `RegisterDeviceEvent`, `RemoveDeviceEvent` and `GrantVaultAccessEvent` are missing

Reply:
Fixed in a3fec58ae9dfa2c41a776279e8ae583072a4b1b1.